### ISSUE TITLE
Translations: key correct?

### ISSUE THREAD

**Issue by rouletout** [theCrag/website#2387]
The name of the key says Deep Water but the text is Rock Climbing - is this correct?
![screen shot 2016-09-18 at 22 34 11](https://cloud.githubusercontent.com/assets/19162952/18616056/32380ec2-7df0-11e6-8e3d-1d559a8e91b0.png)

**Comment by scd** [comment]
This is intentional for some of the minor styles.

**Comment by brendanheywood** [comment]
The only place this is currently used is in the page title and facebook preview:

![image](https://cloud.githubusercontent.com/assets/187449/18619815/331db036-7e49-11e6-8fc6-8e988ece186c.png)

This probably could / should be changed to "Deep water soloing" although the way we use it this may need to be split into two keys, one for the page title where it is used standalone, and one for the preview where it is used as part of a sentence (and could just be that full sentence)

**Comment by scd** [comment]
I have updated this key. I am happy to see how this goes and make other changes as necessary.

**Comment by brendanheywood** [comment]
Just re-tested this. The example given isn't great as it's got mixed climbing DWS and bouldering, none of which is predominant so it defaults back to the generic 'Rock climbing' label. This one in vietnam is all DWS so gets the right string:

https://www.thecrag.com/climbing/vietnam/ha-long-bay/area/27503016

Google was showing the old label but I think that's just a crawling schedule thing and it will sort itself out. However just out of curiosity tailed the access log, I told google to recrawl that page, seconds later I saw the bot hit that exact page, I reloaded the search and it was fixed:

![image](https://cloud.githubusercontent.com/assets/187449/22328401/24b6b704-e410-11e6-9746-8b44a5e22872.png)

Damn! I'm impressed google :)
